sklearn/metrics/cluster/tests/test_common.py

```diff
@@ -232,3 +239,88 @@ def test_returned_value_consistency(name):
 
     assert isinstance(score, float)
     assert not isinstance(score, (np.float64, np.float32))
+
+
+def check_array_api_metric(
```

Comment:
Would importing `check_array_api_metric` from `test_common.py` work? I see it's got redundant 'sample_weight' and multioutput stuff but the logic seems to be there to ignore this if not relevant.

Reply:
In this case, where `cluster` is a submodule in `metrics`, I think I would prefer to avoid code duplication. The second reviewer can weigh in though.